Monitoring/alertmanager/templates

sfawcett123 opened the pull request:
Provided a set of standard templates to make the alerts in slack neat, and provided the ability to pass a number of variables to the alert manager, which permitted the use of links on the slack message.

sfawcett123 committed:
Initial Changes (working)

sfawcett123 committed:
Configuration cleaned up

sfawcett123 committed:
Repeat period set to 1 hour

saliceti commented on terraform/modules/alertmanager/input.tf:
```diff
@@ -4,6 +4,17 @@ variable monitoring_space_id {}
+variable slack_channel { default = "" }
+
+locals {
+  map_variables = {
```
Can we make the name more explicit?
Like config_variables, template_variables, alertmanager_variables...

saliceti commented on terraform/modules/alertmanager/config/slack.tmpl:
```diff
@@ -0,0 +1,86 @@
+# This builds the silence URL.  We exclude the alertname in the range
```
As this was copied from an existing file, can you add a link to the original and credit them?

sfawcett123 committed:
Resolved Colins Comments

sfawcett123 committed:
Update prometheus.yml.tmpl

Made standard 'Prometheus' as it is backward compatible and works with provided dashboards
